ppx_test/with-lwt/ppx_cstruct_and_lwt.ml

```diff
@@ -10,5 +10,5 @@ type foo64 =
   [@@uint64_t]
 ]
 
-let%lwt foo = Lwt.return ()
+let foo = Lwt_main.run (Lwt.return ())
```

Comment:
Why change this? That use of `let%lwt` was there in the test suite was there to check that ppx_cstruct and ppx_lwt work together.

Reply:
The recent version of ppx_lwt gives the warning below,
```
File "ppx_test/with-lwt/ppx_cstruct_and_lwt.ml", line 13, characters 0-27:
Error (warning 22): let%lwt should not be used at the module item level.
Replace let%lwt x = e by let x = Lwt_main.run (e)
```
I have now modified the test to use ```let%lwt``` inside rather than at the module toplevel to get around the warning.

Reply:
Good call; there should be no interop problems with modern ppx, but there were in the past, so its good to keep the test around.